### PR TITLE
chore: update PR template to align it with tools repo

### PR DESCRIPTION
**Pull request by jsomsanith**
**What is the problem this PR is trying to solve?**
PR template has changed over Talend projects, decided on the last TAB.

**What is the chosen solution to this problem?**
Replace the old PR template that contains almost the same info

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,25 +1,20 @@
...
-
-**What is the current behavior?** (You can also link to an open issue here)
-
+**What is the problem this PR is trying to solve?**
 
+**What is the chosen solution to this problem?**
 
-**What is the new behavior?**
+**Link to the JIRA issue**
+e.g. https://jira.talendforge.org/browse/TPSVC-XXX
```
Comment (jmfrancois) [inline .github/PULL_REQUEST_TEMPLATE.md]: we do not have jira for UI

Comment (jsomsanith) [inline .github/PULL_REQUEST_TEMPLATE.md]: Removed

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://425.talend.surge.sh)

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (jsomsanith) [inline .github/PULL_REQUEST_TEMPLATE.md]: + I removed old templates in subfolders since your review

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://425.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://425.talend.surge.sh)

**Pull request merged**